frontend/src/scenes/notebooks/AddToNotebook/DraggableToNotebook.tsx

```diff
@@ -81,6 +80,7 @@ export function DraggableToNotebook({
             <FlaggedFeature flag={FEATURE_FLAGS.NOTEBOOKS} fallback={children}>
                 <span
                     className={clsx('DraggableToNotebook', className, isDragging && 'DraggableToNotebook--dragging')}
+                    draggable={elementProps.onDragEnd ? true : false}
```

Comment:
Is this relevant to the docs panel?

Reply:
This is only semi related. I was trying to select a link and I couldn't, because we override the draggable. I can pull this out to a different PR